docs/UserGuide.md

```diff
@@ -225,7 +228,71 @@ If you accidentally cancel an order, you can still revert its status!
 
 This tutorial will teach you the rewards system in LoyaltyLift and how you can apply it to boost your business' customer loyalty.
 
-    SECTION TODO
+#### The Tier System
+{: .no_toc}
+
+In LoyaltyLift, there are three tiers with their associated point threshold.
+* Bronze - 1000 points
+* Silver - 5000 points
+* Gold - 10000 points
+
+<div markdown="block" class="alert alert-info">
+LoyaltyLift assumes that your tier system will follow a similar style of Bronze, Silver and Gold.
+
+The point threshold above means that if a customer has cumulative points equal to or greater than that threshold, he will be part of that tier.
+For instance, a customer with 5500 cumulative points will be in the Silver tier as he is above the Silver tier point threshold
+but below the Gold tier point threshold. 
+
+</div>
+
+Before we start, head over to [_Add your first customer and order_](#add-your-first-customer-and-order) to include _Lyndon Edwards_ and the _Banana Cake_ order.
+However, do not clear the application data as we will be using the customer for this tutorial.
+
+Rewarding your customers points is an effortless process on LoyaltyLift using 
+[`setpoints`](#setting-reward-points-for-a-customer--setpoints) or 
+[`addpoints`](#addingsubtracting-points-for-a-customer--addpoints).
+
+You would like to add points to _Lyndon Edwards_ for his _Banana Cake_ order.
```

Comment:
Saying "you would like to add points..." sounds weird, feels kind of imposing.

Maybe something like "Let's add points..." or "Let's see how you can add points...", or "In this scenario, we would like to add points..." would be more suitable?

There are a few "you would like to" sprinkled here and there also sounds a little forceful (but the usage for "Perhaps, you would like to..." on line 267 sounds fine as you started with a "Perhaps" so maybe you could do that for other parts too)

Reply:
that is true LOL, i didnt catch that while trying to format the tutorial oops